scripts/run_backend_tests.py

```diff
@@ -603,7 +603,39 @@ def main(args: Optional[List[str]] = None) -> None:
     if parsed_args.generate_coverage_report:
         subprocess.check_call([sys.executable, '-m', 'coverage', 'combine'])
         report_stdout, coverage = check_coverage(True)
+        print('')
+        print(
+            '+----------------------------------------------------------------+'
+            )
+        print(
+            '|-------- '
+            'SUMMARY OF THE FILES WITH INCOMPLETE COVERAGE '
+            '---------|'
+            )
+        print(
+            '+----------------------------------------------------------------+'
+            )
+        print('')
         print(report_stdout)
+        if coverage != 100:
+            print(
+            'WARNING: Backend test coverage is below 100%.\n'
```

Comment:
Indent this and the next 2 lines by 4, since you're creating a new "scope" with the open parens. Ditto for the next two print statements.

Reply:
@seanlip i have made these change in new commit. But waiting for your reply for [this](https://github.com/oppia/oppia/pull/19886#discussion_r1527197909) comment . so that I can make commit having all the changes.